Clayhold Design

jclayhold-sri opened the pull request:
OK, maybe this will work...

lukasc-ubc commented:
Please fix:

Error: Cell bounding box / extent (1258.144, 410.0) is larger than the maximum size of 605.0 X 410.0 microns
1
Errors detected: EBeam_jclayhold.gds, 1 errors.

jclayhold-sri commented:
OK, no worries. I found a duplicated waveguide (now deleted). Found that my floor plan box was drawn in a way that was offset in X (didn't know that was trouble!) so the bounding box was oversized. Selected everything and moved it to (0,0) which I think is better.  But...now out of time! Submissions are closed!  But, no worries. I've still learned 99% of what I wanted, so...all good!

omidesml commented:
There are overlayed waveguides but I am approving this since I think it'll still work although won't be optimum
